ui/lib/core/addon/components/form-field-groups.hbs

```diff
@@ -3,8 +3,8 @@
     {{#if (or (not @renderGroup) (and @renderGroup (eq group @renderGroup)))}}
       {{#if (eq group "default")}}
         {{#each fields as |attr|}}
-          {{! template-lint-configure simple-unless "warn" }}
-          {{#unless (and (not-eq @mode "create") (eq attr.name "name"))}}
```

Comment:
Thanks for cleaning this up 😵 
`if (!(mode !== "create" && attr.name === "name"))`

I wonder if we should call out `isNew` on the component model as something that's only present on models from the search select 🤔 Not sure we have any documentation about what this component expects from the model so that might not be worth doing on this PR. A potential motivation to move some of these to typescript though 👀